service/src/test/kotlin/no/nav/su/se/bakover/service/søknadsbehandling/SøknadsbehandlingTestUtils.kt

```diff
@@ -68,21 +71,21 @@ internal fun createSøknadsbehandlingService(
 ).apply { addObserver(observer) }
 
 internal data class SøknadsbehandlingServiceAndMocks(
-    val søknadsbehandlingRepo: SøknadsbehandlingRepo = mock(),
-    val utbetalingService: UtbetalingService = mock(),
-    val oppgaveService: OppgaveService = mock(),
-    val søknadService: SøknadService = mock(),
-    val personService: PersonService = mock(),
+    val søknadsbehandlingRepo: SøknadsbehandlingRepo = defaultMock(),
```

Comment:
Får håpe dette ikke fører til for mange refaktoreringer da ;)

Reply:
Hva tenker du på? 😛

Reply:
Muligens at dersom vi i stor grad bytter ut mock med defaultMock så fører det til at vi må spesifisere en del mocks vi har slurva på før :)

Reply:
Ja, det stemmer nok